Flint/src/main/java/flint/FlintConfiguration.java

```diff
@@ -1,5 +1,25 @@
 package flint;
 
+import java.io.IOException;
+import java.io.RandomAccessFile;
+import java.util.Collection;
+
+import com.github.javaparser.ast.CompilationUnit;
+
 public abstract class FlintConfiguration {
-  
+  CompilationUnit astRoot;
+  RandomAccessFile inputFile;
+  Collection<FlintRule> rules;
+  Collection<LintFailure> failures;
+
+  public FlintConfiguration(CompilationUnit astRoot, RandomAccessFile inputFile);
```

Comment:
In terms of my implementation of initiating object with file path, I not sure if it supports objects that needs a constructor. Will Check

Related to #7

Reply:
@jessica60933 The constructor should take no arguments. The Adapter is instantiating the Config object, and the Driver is handling parsing, so `runChecks()` should take these arguments.

Reply:
Yeah this makes more sense to me, as I was confused too when we talk about Adapter instantiating the Config object even before driver parsing the file.